std/math/package.d

```diff
@@ -229,7 +230,7 @@ version (D_HardFloat)
     version (IeeeFlagsSupport) version = FloatingPointControlSupport;
 }
 
-version (StdUnittest) private
+version (StdUnittest) package(std.math)
```

Comment:
The `version (StdUnittest)` here causes the failure of circleci`. This was the reason, why I removed it. I don't know now, if I should change anything or not...

Reply:
I haven't checked, but it sounds like the test is compiled with `-version=StdUnittest`, but linked to the release library?

The `-allinst` flag should force the symbol to be emitted, though perhaps the better thing to do would be to replace `equalsDigit` with `isClose` in a prequel PR.

Reply:
This applies also to several other functions too, e.g. the trigonometric ones. I'd suggest to make this temporary `package(std.math)` (without version unittest) and when the splitting is done, go for `isClose`? (What troubles me, is the thought of having a large copy & paste here, which needs to be fixed after such a prequel PR. It's easy to do mistakes (like dropping stuff) which neither git notices nor the test suite.)

Reply:
(And yes, maybe it would be better, if circleci would be fixed.)

Reply:
Ok

Reply:
I mean, I'm not particularly thrilled to introduce a new symbol into the library that is only used by the testsuite, but if we are quick to remove `equalsDigit`, then fine.

Reply:
Well, it's still not public, only package. Shouldn't hurt too much.